### ISSUE TITLE
segfault in udrcx/test_ucp_rma.nonblocking_put_nbi_flush_ep/1 on arm

### ISSUE THREAD

**Issue by yosefe** [openucx/ucx#1486]
http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1592/label=ARM_test_node,worker=0/console
```
12:02:51 [----------] 8 tests from udrcx/test_ucp_rma
12:02:51 [ RUN      ] udrcx/test_ucp_rma.nonblocking_put_nbi_flush_ep/1
12:02:51 [cavium:29398:3] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x9)
13:23:00 Build timed out (after 100 minutes). Marking the build as failed.
13:23:02 Build was aborted
13:23:02 make: *** [test] Terminated
```

**Comment by yosefe** [comment]
related failure  http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1598/label=ARM_test_node,worker=3/console:

```
15:27:08 [ RUN      ] self/test_ucp_tag_xfer.send_contig_recv_contig_unexp_sync_rndv_truncated/0
15:27:08 [       OK ] self/test_ucp_tag_xfer.send_contig_recv_contig_unexp_sync_rndv_truncated/0 (56 ms)
15:27:08 [ RUN      ] self/test_ucp_tag_xfer.contig_unexp_sync/0
15:27:08 [     INFO ] 0 50x10^1 50x10^2 50x10^3 [cavium:74845:3] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x9)
...
(gdb) bt
#0  0x000003ffb6fd16e8 in sched_yield () from /usr/lib64/libc.so.6
#1  0x000003ffb751d380 in spin_acquire_lock (sl=<optimized out>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:1888
#2  ucm_dlfree (mem=mem@entry=0x24c6b880) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:4704
#3  0x000003ffb7515acc in ucm_mem_free (size=<optimized out>, ptr=0x24c6b880) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:238
#4  ucm_free_impl (debug_name=0x3ffb75206d8 "free", orig_free=0x40eff0 <free@plt>, ptr=0x24c6b880) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:249
#5  ucm_free (ptr=0x24c6b880, caller=<optimized out>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:307
#6  0x000003ffb6f8d7d8 in free () from /usr/lib64/libc.so.6
#7  0x000003ffb7647220 in ucs_free (ptr=0x24c6b880) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/debug/memtrack.c:256
#8  0x000003ffb7645bd4 in ucs_debug_cleanup () at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/debug/debug.c:1224
#9  0x000003ffb7645c40 in ucs_error_signal_handler (signo=11, info=0x3ffb6e937a0, context=<optimized out>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/debug/debug.c:957
#10 <signal handler called>
#11 0x000003ffb751d530 in ucm_dlfree (mem=mem@entry=0x24cf2a40) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:4776
#12 0x000003ffb7515acc in ucm_mem_free (size=<optimized out>, ptr=0x24cf2a40) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:238
#13 ucm_free_impl (debug_name=0x3ffb75206d8 "free", orig_free=0x40eff0 <free@plt>, ptr=0x24cf2a40) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:249
#14 ucm_free (ptr=0x24cf2a40, caller=<optimized out>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:307
#15 0x000003ffb6f8d7d8 in free () from /usr/lib64/libc.so.6
#16 0x000003ffb7647220 in ucs_free (ptr=0x24cf2a40) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/debug/memtrack.c:256
#17 0x000003ffb764985c in ucs_stats_node_remove (make_inactive=0, node=0x24d89d70) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/stats/stats.c:153
#18 ucs_stats_clean_node_recurs (node=node@entry=0x3ffb7741bb8 <ucs_stats_context+88>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/stats/stats.c:553
#19 0x000003ffb764a278 in ucs_stats_cleanup () at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../src/ucs/stats/stats.c:588
#20 0x00000000005143ec in uct_test::stats_activate (this=this@entry=0x24bcb230) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/uct/uct_test.cc:138
#21 0x00000000006c92f8 in init (this=0x24bcb230) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/uct/test_rc.h:122
#22 test_dc_flow_control_stats::init (this=0x24bcb230) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/uct/test_dc.cc:492
#23 0x0000000000469774 in ucs::test_base::SetUpProxy (this=0x24bcb248) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/test.cc:167
#24 0x0000000000461c24 in HandleSehExceptionsInMethodIfSupported<testing::Test, void> (location=0x712f20 "SetUp()", method=<optimized out>, object=0x24bcb230)
    at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#25 testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void> (object=object@entry=0x24bcb230, method=<optimized out>, location=location@entry=0x712f20 "SetUp()")
    at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#26 0x0000000000456330 in testing::Test::Run (this=this@entry=0x24bcb230) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3630
#27 0x00000000004564ac in testing::TestInfo::Run (this=0x24b72e30) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3812
#28 0x000000000045661c in testing::TestCase::Run (this=0x24b72f50) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3930
#29 0x000000000045aa2c in testing::internal::UnitTestImpl::RunAllTests (this=this@entry=0x24730ee0) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:5802
#30 0x000000000045ad48 in RunAllTests (this=0x24730ee0) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:5719
#31 HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=<optimized out>, method=<optimized out>, object=0x24730ee0)
    at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#32 HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=0x713408 "auxiliary test code (environments or event listeners)", 
    method=(bool (testing::internal::UnitTestImpl::*)(testing::internal::UnitTestImpl * const)) 0x45ac80 <testing::internal::UnitTestImpl::RunAllTests()>, object=0x24730ee0)
    at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#33 testing::UnitTest::Run (this=<optimized out>) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest-all.cc:5416
#34 0x0000000000413c68 in RUN_ALL_TESTS () at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/gtest.h:20059
#35 main (argc=1, argv=0x3ffda8a94a8) at /scrap/jenkins/workspace/hpc-ucx-pr/label/hpc-arm-04/worker/3/contrib/../test/gtest/common/main.cc:79
```

**Comment by yosefe** [comment]
another one on cavium http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1598/label=ARM_test_node,worker=3/console
```
15:27:08 [ RUN      ] self/test_ucp_tag_xfer.send_contig_recv_contig_unexp_sync_rndv_truncated/0
15:27:08 [       OK ] self/test_ucp_tag_xfer.send_contig_recv_contig_unexp_sync_rndv_truncated/0 (56 ms)
15:27:08 [ RUN      ] self/test_ucp_tag_xfer.contig_unexp_sync/0
15:27:08 [     INFO ] 0 50x10^1 50x10^2 50x10^3 [cavium:74845:3] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x9)
...
(gdb) bt
#0  0x000003ff7aac16e8 in sched_yield () from /lib64/libc.so.6
#1  0x000003ff7b04b7f4 in spin_acquire_lock (sl=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:1888
#2  ucm_dlmalloc (bytes=bytes@entry=2056) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:4576
#3  0x000003ff7b04e59c in ucm_dlmemalign (alignment=<optimized out>, bytes=bytes@entry=2056) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:5267
#4  0x000003ff7b046538 in ucm_malloc_impl (debug_name=0x3ff7b0506f8 "malloc", size=2056) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:226
#5  ucm_malloc (size=2056, caller=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:267
#6  0x000003ff7aa7d354 in malloc () from /lib64/libc.so.6
#7  0x000003ff7b173bd4 in ucs_debug_backtrace_create () at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucs/debug/debug.c:298
#8  0x000003ff7b174588 in ucs_debug_show_innermost_source_file (stream=0x3ff7ab70f40 <_IO_2_1_stderr_>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucs/debug/debug.c:498
#9  0x000003ff7b175398 in ucs_handle_error (error_type=error_type@entry=0x3ff7b20b708 "address not mapped to object", message=message@entry=0x3ff7b20b3c8 "Caught signal %d (%s: %s%s)")
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucs/debug/debug.c:1007
#10 0x000003ff7b175704 in ucs_debug_handle_error_signal (signo=signo@entry=11, cause=0x3ff7b20b708 "address not mapped to object", fmt=fmt@entry=0x3ff7b20b7f8 " at address %p")
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucs/debug/debug.c:951
#11 0x000003ff7b175e78 in ucs_error_signal_handler (signo=11, info=0x3ff7a9837a0, context=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucs/debug/debug.c:974
#12 <signal handler called>
#13 tmalloc_large (m=0x3ff7b0709f0 <_gm_>, nb=864) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:4478
#14 ucm_dlmalloc (bytes=bytes@entry=854) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:4636
#15 0x000003ff7b04e59c in ucm_dlmemalign (alignment=<optimized out>, bytes=bytes@entry=854) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/ptmalloc286/malloc.c:5267
#16 0x000003ff7b046538 in ucm_malloc_impl (debug_name=0x3ff7b0506f8 "malloc", size=854) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:226
#17 ucm_malloc (size=854, caller=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../src/ucm/malloc/malloc_hook.c:267
#18 0x000003ff7aa7d354 in malloc () from /lib64/libc.so.6
#19 0x000003ff7ad2fd9c in operator new(unsigned long) () from /lib64/libstdc++.so.6
#20 0x00000000005b6864 in allocate (this=<optimized out>, __n=854) at /usr/include/c++/4.8.2/ext/new_allocator.h:104
#21 _M_allocate (this=<optimized out>, __n=854) at /usr/include/c++/4.8.2/bits/stl_vector.h:168
#22 _M_create_storage (__n=854, this=0x3ffd61aaf88) at /usr/include/c++/4.8.2/bits/stl_vector.h:181
#23 _Vector_base (__a=..., __n=854, this=0x3ffd61aaf88) at /usr/include/c++/4.8.2/bits/stl_vector.h:136
#24 vector (__a=..., __value=<optimized out>, __n=854, this=0x3ffd61aaf88) at /usr/include/c++/4.8.2/bits/stl_vector.h:297
#25 test_ucp_tag_xfer::test_xfer_contig (this=this@entry=0x414cbbb0, size=854, expected=expected@entry=false, sync=sync@entry=true, truncated=truncated@entry=false)
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/ucp/test_ucp_tag_xfer.cc:290
#26 0x00000000005b3808 in test_ucp_tag_xfer::test_xfer (this=<optimized out>, func=<optimized out>, expected=expected@entry=false, sync=sync@entry=true, truncated=truncated@entry=false, this=<optimized out>, 
    this=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/ucp/test_ucp_tag_xfer.cc:137
#27 0x00000000005b3940 in test_ucp_tag_xfer_contig_unexp_sync_Test::test_body (this=<optimized out>)
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/ucp/test_ucp_tag_xfer.cc:513
#28 0x00000000004695ac in run (this=0x414cbbb0) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/test.cc:204
#29 ucs::test_base::TestBodyProxy (this=0x414cbbb0) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/test.cc:230
#30 0x0000000000461c54 in HandleSehExceptionsInMethodIfSupported<testing::Test, void> (location=0x712f60 "the test body", method=<optimized out>, object=0x414cbc08)
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#31 testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void> (object=object@entry=0x414cbc08, method=<optimized out>, location=location@entry=0x712f60 "the test body")
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#32 0x000000000045640c in testing::Test::Run (this=this@entry=0x414cbc08) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3635
#33 0x00000000004564dc in testing::TestInfo::Run (this=0x41314580) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3812
#34 0x000000000045664c in testing::TestCase::Run (this=0x412dafa0) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3930
#35 0x000000000045aa5c in testing::internal::UnitTestImpl::RunAllTests (this=this@entry=0x41020ee0) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:5802
#36 0x000000000045ad78 in RunAllTests (this=0x41020ee0) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:5719
#37 HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=<optimized out>, method=<optimized out>, object=0x41020ee0)
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#38 HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=0x713440 "auxiliary test code (environments or event listeners)", 
    method=(bool (testing::internal::UnitTestImpl::*)(testing::internal::UnitTestImpl * const)) 0x45acb0 <testing::internal::UnitTestImpl::RunAllTests()>, object=0x41020ee0)
    at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#39 testing::UnitTest::Run (this=<optimized out>) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest-all.cc:5416
#40 0x0000000000413c98 in RUN_ALL_TESTS () at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/gtest.h:20059
#41 main (argc=1, argv=0x3ffd61ab488) at /scrap/jenkins/workspace/gh-ucx-pr-2/label/ARM_test_node/worker/3/contrib/../test/gtest/common/main.cc:79
```

**Comment by yosefe** [comment]
similar failure on x86:
```
(gdb) bt
#0  0x00007ffff6b5ca2d in pause () from /usr/lib64/libpthread.so.0
#1  0x00007ffff7adba65 in ucs_debug_freeze () at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../src/ucs/debug/debug.c:728
#2  0x00007ffff7add585 in ucs_error_freeze (message=0x7ffff7ff5d40 "Caught signal 11 (Segmentation fault: address not mapped to object at address (nil))", 
    error_type=0x7ffff7b851ce "address not mapped to object") at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../src/ucs/debug/debug.c:847
#3  ucs_handle_error (error_type=error_type@entry=0x7ffff7b851ce "address not mapped to object", message=message@entry=0x7ffff7b84fa8 "Caught signal %d (%s: %s%s)")
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../src/ucs/debug/debug.c:1011
#4  0x00007ffff7add947 in ucs_debug_handle_error_signal (signo=signo@entry=11, cause=0x7ffff7b851ce "address not mapped to object", fmt=fmt@entry=0x7ffff7b85270 " at address %p")
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../src/ucs/debug/debug.c:951
#5  0x00007ffff7ade0bb in ucs_error_signal_handler (signo=11, info=0x7ffff7ff65f0, context=<optimized out>)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../src/ucs/debug/debug.c:974
#6  <signal handler called>
#7  0x0000000000000000 in ?? ()
#8  0x00000000031e9230 in ?? ()
#9  0x0000000000520810 in test_uct_stats::cleanup (this=0x31e9230)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/uct/test_stats.cc:42
#10 0x000000000046780a in ucs::test_base::TearDownProxy (this=0x31e9248)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/test.cc:185
#11 0x000000000045f243 in HandleSehExceptionsInMethodIfSupported<testing::Test, void> (location=0x72205a "TearDown()", method=<optimized out>, object=<optimized out>)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#12 testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void> (object=object@entry=0x31e9230, method=<optimized out>, location=location@entry=0x72205a "TearDown()")
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#13 0x000000000045323c in testing::Test::Run (this=this@entry=0x31e9230)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3643
#14 0x000000000045333c in testing::TestInfo::Run (this=0x2b88530)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3812
#15 0x000000000045349f in testing::TestCase::Run (this=0x2b5d3f0)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3930
#16 0x0000000000457fb7 in testing::internal::UnitTestImpl::RunAllTests (this=this@entry=0x2a5c730)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:5802
#17 0x00000000004582bb in RunAllTests (this=0x2a5c730) at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:5719
#18 HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=<optimized out>, method=<optimized out>, object=0x2a5c730)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3562
#19 HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool> (location=0x723530 "auxiliary test code (environments or event listeners)", 
    method=(bool (testing::internal::UnitTestImpl::*)(testing::internal::UnitTestImpl * const)) 0x458200 <testing::internal::UnitTestImpl::RunAllTests()>, object=0x2a5c730)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:3598
#20 testing::UnitTest::Run (this=<optimized out>) at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest-all.cc:5416
#21 0x000000000040e9e2 in RUN_ALL_TESTS () at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/gtest.h:20059
#22 main (argc=1, argv=<optimized out>) at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/common/main.cc:79
(gdb) f 9
#9  0x0000000000520810 in test_uct_stats::cleanup (this=0x31e9230)
    at /scrap/jenkins/scrap/workspace/hpc-ucx-pr/label/hpc-test-node2/worker/3/contrib/../test/gtest/uct/test_stats.cc:42
42	        uct_p2p_test::cleanup();
```

**Comment by yosefe** [comment]
fixed by #1499
